safehttp/response_writer_test.go

```diff
@@ -66,6 +67,18 @@ func TestResponseWriterWriteTwicePanic(t *testing.T) {
 				w.Write(safehtml.HTMLEscaped("<h1>Escaped, so not really a heading</h1>"))
 			},
 		},
+		{
+			name: "Call WriteJSON twice",
+			write: func(w *safehttp.ResponseWriter) {
+				data := struct{ Field string }{Field: "myField"}
+				jsonObj, err := json.Marshal(data)
+				if err != nil {
+					t.Fatalf("invalid json object")
+				}
+				w.WriteJSON(jsonObj)
+				w.WriteJSON(jsonObj)
+			},
```

Comment:
When you use `WriteJSON` you send the actual struct, right? The dispatcher will marshal it for you so that isn't needed here. Now you are sending just a string.